src/ProcessService/ProcessServiceUtils.cpp

```diff
@@ -273,40 +278,43 @@ ErrorMessageOr<fs::path> FindSymbolsFilePath(
     }
   }
 
-  std::vector<std::string> error_messages;
-
   for (const auto& search_path : search_paths) {
     ErrorMessageOr<bool> file_exists = orbit_base::FileExists(search_path);
 
     if (file_exists.has_error()) {
-      ORBIT_ERROR("%s", file_exists.error().message());
-      error_messages.emplace_back(file_exists.error().message());
+      std::string not_found_message{absl::StrFormat(
+          "Potential symbols file \"%s\" could not be checked for existence, error: %s",
```

Comment:
I would drop the `, error` parts in these messages (same below) as we always intend that what comes after the colon is a more detailed description of the error.

Reply:
done